README.md

```diff
@@ -294,3 +304,58 @@ class ArrayHelper_CXFileUniqueID_data_level0 {
   2. Run `dart build_test_dylib.dart`.
 
 Run tests from the root of the package with `pub run test`.
+
+## FAQ
+<details><summary>Removing underscores or renaming declarations.</summary>
```

Comment:
This should be a question, it's a FAQ after all.

### Can ffigen remove underscores or rename declarations?

(same for other headers)